packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js

```diff
@@ -44,6 +44,15 @@ skipInNode(describe)('plugin-meetings', () => {
           console.log('CONVERSATION', conversation);
           space = conversation;
         })
+        .then(async () => {
+          const destinationWithType = await alice.webex.meetings.meetingInfo.fetchMeetingInfo(space.url, 'CONVERSATION_URL');
```

Comment:
I see new test is added for cases with type===CONVERSATION_URL and type===undefined, but the code changes in this PR are for the case when type===_LOCUS_ID_. Where is the test for that? Is it already covered by some existing test? 
Also while trying to see if there is an existing test I came across tests that check that `MeetingsUtil.extractDestination` has been called (like here: https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/plugin-meetings/test/unit/spec/meetings/index.js#L450 and more cases further in that file) and `MeetingsUtil.extractDestination` doesn't exist anymore as it was moved into src/meeting-info/util.js so I'm not sure how these tests are passing. I would expect that they need updating, or am I missing something?

Reply:
Going to get handled here https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-259981